torch_xla/csrc/device.cpp

```diff
@@ -81,12 +81,17 @@ bool ShouldUseVirtualDevice() {
   return use_virtual_device;
 }
 
-bool UseVirtualDevice() {
+bool UseVirtualDevice(bool flip_spmd_config) {
```

Comment:
this `flip_spmd_config` is a bit too hacky. we used `static bool` because we don't intend to change this field. If we do intend to change it, it is better to put the static bool in the top of this file and introduce a `SetUseVirtualDevice(bool)`. You can fix that in a follow up pr.

Reply:
Thanks, I adoped this as well as Jon's. We can refactor once again after making the device mutable in the tensor.